runtime/src/bridge.rs

```diff
@@ -257,13 +213,75 @@ impl<T: Trait> Module<T> {
 			.or_else(ensure_root)?;
 		Ok(())
 	}
+
+        fn do_transfer_native(origin: T::Origin, amount: BalanceOf<T>, recipient: Vec<u8>, dest_id: chainbridge::ChainId) -> DispatchResult {
+            let source = ensure_signed(origin)?;
+
+            let token_fee: T::Balance = TokenTransferFee::<T>::get();
+            let currency_token_fee = TryInto::<u128>::try_into(token_fee).map_err(|_| "Token fee not convertible to currency")?
+                                                     .try_into().map_err(|_| "Token fee not convertible to currency")?;
```

Comment:
Is this `try_into` duplicate?

Reply:
Alas not redundant. We first convert the T::Balance into a u128, and then from that into a Currency::Balance. I could not figure out how to get the compiler to believe that those two balances were the same type, so we have this unfortunate dance.